repoclosure/yum.conf

```diff
@@ -0,0 +1,160 @@
+baseurl=https://yum.puppetlabs.com/el/7/PC1/x86_64/
+
+[el7-puppet-5]
+name=el7-puppet-5
+baseurl=https://yum.puppetlabs.com/puppet5/el/7/x86_64/
+
+[el7-foreman-rails-nightly]
+name=Foreman Rails nightly EL7
+baseurl=https://yum.theforeman.org/rails/foreman-nightly/el7/x86_64
+
+# Used as lookaside repos for layered repos (plugins)
+[el7-foreman-nightly]
+name=Foreman nightly EL7
+baseurl=https://yum.theforeman.org/nightly/el7/x86_64
+
+[el7-foreman-plugins-nightly]
+name=Foreman Plugins nightly EL7
+baseurl=https://yum.theforeman.org/plugins/nightly/el7/x86_64
+
+# Katello repos for lookaside
+[el7-katello-nightly]
+name=Katello nightly EL7
+baseurl=https://fedorapeople.org/groups/katello/releases/yum/nightly/katello/el7/x86_64
+
+[el7-katello-pulp-nightly]
+name=Katello Pulp nightly EL7
+baseurl=https://repos.fedorapeople.org/repos/pulp/pulp/stable/2.15/7Server/x86_64/
+
+[el7-katello-candlepin-nightly]
+name=Katello Candlepin nightly EL7
+baseurl=https://fedorapeople.org/groups/katello/releases/yum/nightly/candlepin/el7/x86_64
+
+[el7-katello-client-nightly]
+name=Katello nightly EL7
+baseurl=https://fedorapeople.org/groups/katello/releases/yum/nightly/client/el7/x86_64
+
+
+[el6-base]
+name=BaseOS
+enabled=1
+mirrorlist=http://mirrorlist.centos.org/?release=6&arch=x86_64&repo=os
+failovermethod=priority
+
+[el6-updates]
+name=updates
+enabled=1
+mirrorlist=http://mirrorlist.centos.org/?release=6&arch=x86_64&repo=updates
+failovermethod=priority
+
+[el6-extras]
+name=extras
+mirrorlist=http://mirrorlist.centos.org/?release=6&arch=x86_64&repo=extras
+
+[el6-epel]
+name=epel
+mirrorlist=http://mirrors.fedoraproject.org/mirrorlist?repo=epel-6&arch=x86_64
+failovermethod=priority
+
+[el6-puppet-4]
+name=el6-puppet-4
+baseurl=http://yum.puppetlabs.com/el/6/PC1/x86_64/
+
+[el6-puppet-5]
+name=el6-puppet-5
+baseurl=https://yum.puppetlabs.com/puppet5/el/6/x86_64/
+
+[el6-subscription-manager]
+name=el6-subscription-manager
+baseurl=https://copr-be.cloud.fedoraproject.org/results/dgoodwin/subscription-manager/epel-6-x86_64/
+
+[el6-qpid]
+name=el6-qpid
+baseurl=https://copr-be.cloud.fedoraproject.org/results/ndp/qpid/epel-6-x86_64/
+
+[el6-katello-client-nightly]
+name=Katello nightly EL6
+baseurl=https://fedorapeople.org/groups/katello/releases/yum/nightly/client/el6/x86_64
+
+[el5-base]
+name=BaseOS
+enabled=1
+baseurl=http://vault.centos.org/5.11/os/x86_64/
+
+[el5-epel]
+name=epel
+baseurl=https://pubmirror2.math.uh.edu/fedora-buffet/archive/epel/5/x86_64/
+
+[el5-puppet-4]
+name=el5-puppet-4
+baseurl=http://yum.puppetlabs.com/el/5/PC1/x86_64/
+
+[el5-puppet-5]
+name=el5-puppet-5
+baseurl=https://yum.puppetlabs.com/puppet5/el/5/x86_64/
+
+[el5-subscription-manager]
+name=el5-subscription-manager
+baseurl=https://repos.fedorapeople.org/candlepin/subscription-manager/epel-5/x86_64/
+
+[el5-katello-client-nightly]
+name=Katello nightly EL5
+baseurl=https://fedorapeople.org/groups/katello/releases/yum/nightly/client/el5/x86_64
+
+[f27-base]
```

Comment:
Do we need equivalent f28 configs here?

Reply:
Not yet -- f28 does not currently work due to a Koji/mock bug.